VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -291,7 +298,9 @@ public void onEventMainThread(DiscussionThreadPostedEvent event) {
             // move the ListView's scroll to that newly added post's position
             discussionPostsListView.setSelection(i);
             // In case this is the first addition, we need to hide the no-item-view
-            setNoResultView(null);
+            setScreenStateUponResult(null);
+            // And show the filter and sort spinners
+            spinnersContainerLayout.setVisibility(View.VISIBLE);
```

Comment:
It looks like this should already be taken care of by the `setScreenStateUponResult()` implementation.